src/Plugin.cpp

```diff
@@ -75,15 +78,16 @@ void Plugin::init() {
           "There is no Anchor \"" + settings.first + "\" defined in the settings.");
     }
 
-    mGuiManager->getSideBar()->callJavascript(
-        "add_celestial_body", anchor->second.mCenter, settings.second.mIcon);
+    mGuiManager->getGui()->callJavascript("CosmoScout.call", "flyto", "addCelestialBody",
```

Comment:
Also in this plugin I would prefer the `callJavascript("CosmoScout.flyto.addCelestialBody",...)` syntax.

Reply:
Done in https://github.com/cosmoscout/csp-fly-to-locations/pull/3/commits/98da0e6454a8e3c4d8b1518bb380e17ddc80eefb